Exercises-4-OOP/src/exercisesonclasses/Rectangle.java

```diff
@@ -0,0 +1,47 @@
+package exercisesonclasses;
+//Ex: The Rectangle Class.
+
+public class Rectangle {
+	private float length = 1.0f;
+	private float width = 1.0f;
+
+	public Rectangle() {
+
+	}
```

Comment:
```suggestion
	public Rectangle() {}
```
When you are writing an empty constructor, you can just write it like this to make it shorter.
Please make the change in other classes as well.

Reply:
Done.